src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +92,70 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={query}
+              onChange={handleMovieQuery}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {error
+          && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
+
+          {sameMovie
+          && (
+            <p className="help is-danger">
+              This movie has already been added to the list
+            </p>
+          )}
+
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
-            >
-              Find a movie
-            </button>
-          </div>
-
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
+              className={classNames(
+                'button',
+                'is-light',
+                {
+                  'is-loading': isLoading,
+                },
```

Comment:
If line length allows - better always write this way 👇 Code would be more readable.
```suggestion
                { 'is-loading': isLoading },
```